deisctl/deisctl.go

```diff
@@ -32,6 +32,7 @@ Commands, use "deisctl help <command>" to learn more:
   install           install components, or the entire platform
   uninstall         uninstall components
   list              list installed components
+  list-machines     Enumerate the current hosts in the cluster
```

Comment:
We bickered amongst ourselves and found `machines` to be the best candidate for v1. We can change it to `nodes` in v2 if it's too confusing (and as @mboersma pointed out, we can also just make it an alias of `machines`, too).